File: hello-world/python/receive.py
#!/usr/bin/env python
import pika

connection = pika.BlockingConnection(pika.ConnectionParameters(host='localhost'))
channel = connection.channel()

# queue_declare is idempotent - so, it will not
# create the same queue again if has been existed.
channel.queue_declare(queue='hello')

def callback(ch, method, properties, body):
    # use decode to remove b'' (bytes literal)
    print(" [x] Received %r" % body.decode('UTF-8'))

# Receive message
channel.basic_consume(
    callback,
    queue='hello',
    no_ack=True
)

print(' [x] Waiting for messages. To exit press CTRL+C')
channel.start_consuming()

File: hello-world/javascript/receive.js
#!/usr/bin/env node
var amqp = require('amqplib/callback_api');

amqp.connect('amqp://localhost', (err, conn) => {
    conn.createChannel((err, ch) => {
        let queue_name = 'hello';

        ch.assertQueue(queue_name, { durable: false });

        console.log(" [x] Waiting for message in %s. To exit press CTRL+C", queue_name);

        // Receive message
        ch.consume(queue_name, (msg) => {
            console.log(" [x] Received %s", msg.content.toString());
        }, { noAck: true });
    });
});